pango/src/glyph_item_iter.rs

```diff
@@ -41,4 +41,23 @@ impl GlyphItemIter {
             }
         }
     }
+
+    pub fn start_char(&self) -> i32 {
```

Comment:
this commit shouldn't be part of this PR :)

Reply:
I made some errors while rebasing and squashing probably

Reply:
That's funny... I can't see this change locally:
```
❯ git diff --stat upstream/master
 glib/src/gobject/flags.rs |   6 +++++
 glib/src/param_spec.rs    | 266 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++-------------
 2 files changed, 253 insertions(+), 19 deletions(-)
 ```
 Maybe another force push will fix it

Reply:
You have that change in a separate commit now, so getting rid of it should be just a matter of dropping it from `git rebase origin/master -i`.

Reply:
I don't understand why the change didn't show up while doing `git diff upstream/master`, but indeed, doing `git rebase -i upstream/master` auto-removed it (By the way, it didn't show up even in the interactive rebase... Bah)